src/robot/result/xmlelementhandlers.py

```diff
@@ -240,13 +240,31 @@ def start(self, elem, result):
         return result.body.create_return()
 
 
+@ElementHandler.register
+class ContinueHandler(ElementHandler):
+    tag = 'continue'
+    children = frozenset(('status', 'msg'))
+
+    def start(self, elem, result):
+        return result.body.create_continue()
+
+
+@ElementHandler.register
+class BreakHandler(ElementHandler):
+    tag = 'break'
+    children = frozenset(('status', 'msg'))
+
+    def start(self, elem, result):
+        return result.body.create_break()
+
```

Comment:
In addition to creating these handlers for `<break>` and `<continue>`, they need to be listed in `children` for `IterationHandler` and `BranchHandler`. After this change `BREAK` and `CONTINUE` should be visible in log as well.

`<return>` is also used with `KeywordHandler`, but these statements aren't usable directly under keywords so that shouldn't be needed.